data_sources/sysmon_eventid_13.yml

```diff
@@ -101,6 +101,19 @@ fields:
 - timestartpos
 - user_id
 - vendor_product
+output_fields:
+- action 
+- dest 
+- process_guid 
```

Comment:
lets add process_name, process_path

Reply:
As discussed, we can't add it because it is not part of the CIM data model.